src/compute/src/logging/compute.rs

```diff
@@ -296,22 +312,26 @@ pub(super) fn construct<A: Allocate + 'static>(
         let mut packer = PermutedRowPacker::new(ComputeLog::PeekCurrent);
         let peek_current = peek.as_collection().map({
             let mut scratch = String::new();
-            move |datum| {
+            move |(typ, datum)| {
                 packer.pack_slice(&[
                     Datum::Uuid(datum.uuid),
                     Datum::UInt64(u64::cast_from(worker_id)),
                     make_string_datum(datum.id, &mut scratch),
+                    typ.name().into(),
```

Comment:
I can swap these to use `Datum::String` directly to make that clear.

While `make_string_datum` may avoid an allocation in the common case, it's still inherently less efficient since it does an extra data copy, where this code just threads around a pointer. If you still prefer the `to_string` route, I can do this - just want to make sure it's clear why I went this route!

Reply:
Ah, I totally missed that `name` returns a `&str`, not a `String`. No, the `ToString` route doesn't make sense then, all good!